doc_admin/AdminManual.adoc

```diff
@@ -1614,22 +1621,24 @@ dicom.max.size=DICOM_SIZE
 Use the provided startup scripts in order to automatically generate the _dicom.properties_ file from the corresponding _.dist_ file; after generation, optionally adjust the following parameters:
 
 * *DICOM_STORAGE*: This property accepts one of the following:
-** SqlDicomManager: DICOM files are stored in the database
-** FileSystemDicomManager: DICOM files are stored in the file system
+** SqlDicomManager: Image files are stored in the database
+** FileSystemDicomManager: Image files are stored in the file system
 * *OH_PATH_SUBSTITUTE/DICOM_DIR*: if the value of the first parameter is set to "FileSystemDicomManager",
-this property specifies the path where the DICOM files are stored. It is possible to specify a local folder (_data/dicom_storage_ is the default path under the OH installation folder) or to set a shared network folder, useful for LAN - client/server environment. Suggested configuration is to map the network/NFS folder under the _data/dicom_storage_ path with a symbolic link.
-The shared folder needs read/write permissions for uploading DICOM images. Always use slash "/" in the path variable and not backslash "\".
-* *DICOM_SIZE*: maximum allowed size for a dicom/jpg image (the MariaDB/MySQL server setting max_allowed_packet should be set with the same or a larger value).
+this property specifies the path where the image files are stored. It is possible to specify a local folder (_data/image_storage_ is the default path under the OH installation folder) or to set a shared network folder, useful for LAN - client/server environment. Suggested configuration is to map the network/NFS folder under the _data/image_storage_ path with a symbolic link.
```

Comment:
also this one

Reply:
I'm confused; I've replaced "DICOM files" with "image files"; "dicom_storage_" with "image_storage_"; what is left to do?

Reply:
sorry, there was a pending comment, see above https://github.com/informatici/openhospital-doc/pull/180/files#r1028432221